services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/api/rest/routes.py

```diff
@@ -16,4 +16,4 @@ def setup_rest_api(app: FastAPI):
     app.include_router(api_router)
 
     app.add_exception_handler(Exception, handle_errors_as_500)
-    app.add_exception_handler(HTTPException, http_exception_as_json_response)
+    app.add_exception_handler(HTTPException, http_exception_as_json_response)  # type: ignore[arg-type]
```

Comment:
why is the ignore necessary? are you sure you are not ignoring something that changed?

Reply:
fixed together